### DISCUSSION TITLE
Shared deck milestone

### DISCUSSION THREAD

**Discussion by axelboc** [axelboc/anki-ultimate-geography#459] [Show and tell]
The [shared deck](https://ankiweb.net/shared/info/2109889812) has 200 thumbs-up! Just thought it was worth sharing 🙌 

![image](https://user-images.githubusercontent.com/2936402/130657256-60449701-630d-4bcc-a358-4dee26f2b186.png)

**Comment by aplaice** [comment]
Thanks for sharing! It's always nice to read positive feedback for what we do! :D 

<hr/>

It's a shame that some people are asking questions in the reviews, such that we can't answer them.  For instance:

>  Cool deck, but do you have a higher resolution version or svg version?

(In the extraordinarily unlikely case that the person asking the question sees this: we could generate higher resolution images, but it's a trade-off of quality vs. size. Generating higher resolution images for personal use should be straightforward.)

**Reply by axelboc** [reply]
Yeah, I try to create issues when I see feedback we haven't discussed here already.

**Comment by axelboc** [comment]
I remember laughing at this downvote:

![image](https://user-images.githubusercontent.com/2936402/130740841-a34095a6-3f23-43bf-a5b1-63e538029c2e.png)
